extensions/BMO/Extension.pm

```diff
@@ -713,6 +713,18 @@ sub bug_format_comment {
         }
     });
 
+    # link github pull requests and issues
+    push (@$regexes, {
+        match => qr#\b([A-Za-z0-9_\.-]+\/[A-Za-z0-9_\.-]+\#[0-9]+)\b#mx,
```

Comment:
Unfortunately this regular expression is so broad it will be a performance problem.
There are 11,764,265 comments on BMO.

For the short term, we can hard-code a list of projects globally,
and later on I will add a way of linking components to external resources.

I'm updating the bug with instructions on this.